src/Optimizer/Transformer/OptimizeHeroImages.php

```diff
@@ -555,6 +555,7 @@ private function generateImg(HeroImage $heroImage, Document $document)
         $imgElement = $document->createElement(Tag::IMG);
         $imgElement->setAttribute(Attribute::CLASS_, self::SSR_IMAGE_CLASS);
         $imgElement->setAttribute(Attribute::DECODING, 'async');
+        $imgElement->setAttribute(Attribute::FETCHPRIORITY, 'high');
```

Comment:
Thinking about this some more now... perhaps we actually shouldn't automatically add `fetchpriority=high` to all hero images, since there can be multiple. The `fetchpriority=high` attribute should be reserved for the LCP image. So I think instead of automatically adding it to all heros, we should instead only it when the `amp-img > noscript > img` contains `fetchpriority=high`.

Additionally, we should automatically SSR any `amp-img` that has a `noscript > img` with `fetchpriority=high`.

But if an `amp-img` has `data-hero` alone, that wouldn't necessarily warrant adding `fetchpriority=high`.

Reply:
> Thinking about this some more now... perhaps we actually shouldn't automatically add fetchpriority=high to all hero images, since there can be multiple. 

Completely agree here and I have updated it per suggestion.